### ISSUE TITLE
support alt+* keys to operate menus

### ISSUE THREAD

**Issue by akosyakov** [eclipse-theia/theia#7063]
For example alt+f should open a file menu and so on. We should align as usual with VS Code by default. It should be doable for browser target as well (i've checked that google drive can do it)

**Comment by tsmaeder** [comment]
Note that Firefox _does_ open the browser menu when pressing `Alt`. Overriding that behaviour might not be desired.

**Comment by akosyakov** [comment]
Yes, we will need to have what is the sensitive behaviour for different operating system and browsers. Windows for example highlights a letter which should be used together with Alt in the menu name, but Mac does not do it by default. Such things should be considered as well.
